packages/backend/src/app/authentication/authentication.controller.ts

```diff
@@ -27,9 +28,15 @@ export const authenticationController: FastifyPluginAsyncTypebox = async (app) =
     })
 }
 
+const rateLimitOptions: RateLimitOptions = {
+    max: 50,
```

Comment:
I think around 10 should be more than enough for sign in

Reply:
Are you sure about this? multiple devices normally share public IPs, this could make a user unable to sign in because another user sharing the same public IP made a lot of requests.